packages/client/src/Config.ts

```diff
@@ -144,8 +149,9 @@ export interface NetworkPeerDescriptor {
 }
 
 export interface ConnectivityMethod {
```

Comment:
Could this be just an URL as @ptesavol suggested? I.e. `ws://127.0.0.1:40500` instead of 
```
{
    host: 127.0.0.1,
    port: 40500,
    tls: false
}
``` 

Maybe we don't need the object form anywhere if we use URL form?

Reply:
https://linear.app/streamr/issue/NET-1076/configure-and-use-websocket-server-url-as-an-url